_episodes/04-tabular_data.md

```diff
@@ -341,18 +349,18 @@ TYR6_ASP : 10.9934435
 >>     atom_data = atom.split()
 >>     symbol = atom_data[0]
 >>     symbols.append(symbol)
->>     x, y, z = atom_data[1], atom_data[2], atom_data[3]
+>>     x, y, z = atom_data[1:]    # or `x, y, z = atom_data[1], atom_data[2], atom_data[3]`
```

Comment:
We prefer to be as clear to beginners as possible in this material. This assignment will likely confuse those new to programming, so we do not include it. It may be fine to include in the opposite way:

```
x, y, z = atom_data[1], atom_data[2], atom_data[3]   # or x, y, z = atom_data[1:] 
```

or included as a callout

Reply:
I have changed it back to the original document because I think my suggestion would likely cause unnecessary confusion for beginners.